Geometry/TrackerGeometryBuilder/src/TrackerGeomBuilderFromGeometricDet.cc

```diff
@@ -43,8 +43,14 @@ namespace {
 TrackerGeometry* TrackerGeomBuilderFromGeometricDet::build(const GeometricDet* gd,
                                                            const PTrackerParameters& ptp,
                                                            const TrackerTopology* tTopo) {
-  int BIG_PIX_PER_ROC_X = ptp.vpars[2];
-  int BIG_PIX_PER_ROC_Y = ptp.vpars[3];
+  if (ptp.vpars.size() != 6) {
+    edm::LogError("TrackerGeomBuilderFromGeometricDet")
```

Comment:
@ghugo83 - should it be throw? Does it make sense to continue? Because it might throw 6 lines later :-)

Reply:
Hmm here if ptp.vpars.size() != 6, the values for BIG_PIXEL might be wrong / null, which is not nice but not the end of the world (these values are null for now for Phase 2  by the way, while they should be changed at some point).
The rest of the results could still be interesting to get.

But as you prefer :)